samples/highcharts/demo/stellar-chart/demo.js

```diff
@@ -0,0 +1,224 @@
+const colors = ['#6CDDCA', '#C771F3', '#4D90DB', '#FAB776'];
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'bubble',
+        polar: true,
+        height: 600,
+        events: {
+            render() {
+                const chart = this,
+                    pieSeries = chart.series[1];
+                pieSeries.customLabel = fillCenter(100, '1990s-2020s', chart,
+                    pieSeries.customLabel);
+            }
+        }
+    },
+    data: {
+        csv: document.getElementById('csv').innerHTML,
+        seriesMapping: [{
+            name: 0,
+            x: 1,
+            y: 2,
+            z: 3,
+            lightYears: 4,
+            planetMass: 5,
+            stellarMagnitude: 6,
+            discoveryDate: 7
```

Comment:
Custom stuff should be in [`custom`](https://api.highcharts.com/highcharts/series.bubble.data.custom).